packages/trace-viewer/src/snapshotServer.ts

```diff
@@ -103,13 +100,3 @@ declare global {
     showSnapshot: (url: string, point?: Point) => Promise<void>;
   }
 }
-
-function removeHash(url: string) {
```

Comment:
Why dropped this one? We intercept without a hash, but page requests with a hash, so we make sure url matches.

Reply:
Ouch